src/backend/contactRequest/contactRequest.service.ts

```diff
@@ -0,0 +1,30 @@
+import { httpClient } from '@/lib/httpClient'
+import { mailerLiteClient } from '@/lib/mailerliteClient'
+
+export async function addSubscriberToMailerLite(
+  email: string,
+  groupId: string,
+  fields?: Record<string, string>,
+) {
```

Comment:
**nit:** these external lib related methods should be placed in `lib/mailerliteClient.ts`